doc/manual/src/advanced-topics/post-build-hook.md

```diff
@@ -33,12 +33,16 @@ distribute the public key for verifying the authenticity of the paths.
 example-nix-cache-1:1/cKDz3QCCOmwcztD2eV6Coggp6rqc9DGjWv7C0G+rM=
 ```
 
-Then, add the public key and the cache URL to your `nix.conf`'s
-`trusted-public-keys` and `substituters` options:
+Then update [`nix.conf`](../command-ref/conf-file.md) on any machine that will access the cache, adding the cache URL to [`substituters`](../command-ref/conf-file.md#conf-substituters) and the public key to [`trusted-public-keys`](../command-ref/conf-file.md#conf-trusted-public-keys):
```

Comment:
```suggestion
Then update [`nix.conf`](../command-ref/conf-file.md) on any machine that will access the cache.
Add the cache URL to [`substituters`](../command-ref/conf-file.md#conf-substituters) and the public key to [`trusted-public-keys`](../command-ref/conf-file.md#conf-trusted-public-keys):
```

Use short sentences where possible. Please keep the reference links, it makes the source much easier to read and maintain. The links are also easier to maintain in bulk when they are not inlined, i.e. when we rename files. (Search/replace still has to be checked manually until we have automatic link checks.)

See https://github.com/NixOS/nix.dev/blob/master/CONTRIBUTING.md#guides

This is not consistent at all yet, but we're only starting to work on this.